src/containers/Latest.js

```diff
@@ -0,0 +1,348 @@
+import { connect } from 'react-redux'
+import Helmet from 'react-helmet'
+import PropTypes from 'prop-types'
+import querystring from 'querystring'
+import React, { useEffect } from 'react'
+import styled from 'styled-components'
+// components
+import SystemError from '../components/SystemError'
+import Pagination from '../components/Pagination'
+// utils
+import loggerFactory from '../logger'
+import { shallowCloneMetaOfPost } from '../utils/shallow-clone-entity'
+// constants
+import dataLoaderConst from '../constants/data-loaders'
+import siteMeta from '../constants/site-meta'
+// @twreporter
+import twreporterRedux from '@twreporter/redux'
+import mq from '@twreporter/core/lib/utils/media-query'
+import { List } from '@twreporter/react-components/lib/listing-page'
+import { TitleTab } from '@twreporter/react-components/lib/title-bar'
+// lodash
+import forEach from 'lodash/forEach'
+import get from 'lodash/get'
+import map from 'lodash/map'
+import findIndex from 'lodash/findIndex'
+import merge from 'lodash/merge'
+import concat from 'lodash/concat'
+const _ = {
+  forEach,
+  get,
+  map,
+  findIndex,
+  merge,
+  concat,
+}
+// global var
+const { actions, reduxStateFields } = twreporterRedux
+const { fetchPostsByTagListId, fetchLatestPosts, fetchLatestTags } = actions
+const logger = loggerFactory.getLogger()
+
+const Container = styled.div`
+  margin: 0 auto;
+  max-width: 1130px; //hd
+
+  ${mq.desktopOnly`
+    max-width: 922px;
+  `}
+  ${mq.tabletOnly`
+    max-width: 698px;
+  `}
+  ${mq.mobileOnly`
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    max-width: 767px;
+  `}
+`
+
+const TitleTabContainer = styled.div`
+  margin: 64px 0;
+
+  ${mq.tabletOnly`
+    margin: 32px 0;
+  `}
+  ${mq.mobileOnly`
+    margin: 24px 0;
+    width: 86.66666666666667%;
```

Comment:
not so important, but curious about this setting, do we really need so many digits? 😂 what about just use 86.67%

Reply:
Good point 😆
It's just the copy code from previous...
I think we could manage it when the layout need to change.